File: DevExpress VCL/ExpressQuantumGrid 7/Demos/CBuilder/CustomDrawTableViewDemo/CustomDrawTableViewDemoData.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "CustomDrawTableViewDemoData.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxClasses"
#pragma link "cxGridTableView"
#pragma link "cxStyles"
#pragma resource "*.dfm"
TCustomDrawTableViewDemoMainDM *CustomDrawTableViewDemoMainDM;
//---------------------------------------------------------------------------
__fastcall TCustomDrawTableViewDemoMainDM::TCustomDrawTableViewDemoMainDM(TComponent* Owner)
  : TDataModule(Owner)
{
}
//---------------------------------------------------------------------------

File: DevExpress VCL/ExpressQuantumGrid 6/Demos/Delphi/CustomDrawTableViewDemo/CustomDrawTableViewDemoData.pas
unit CustomDrawTableViewDemoData;

interface

uses
  SysUtils, Classes, DB, cxStyles, cxClasses, cxGridTableView, Forms, DBTables;

type
  TCustomDrawTableViewDemoMainDM = class(TDataModule)
    DataBase: TDataBase;
    StyleRepository: TcxStyleRepository;
    tlbCars: TTable;
    dsCars: TDataSource;
    tlbCustomers: TTable;
    dsCustomers: TDataSource;
    tlbOrders: TTable;
    dsOrders: TDataSource;
    GridTableViewStyleSheetDevExpress: TcxGridTableViewStyleSheet;
    cxStyle1: TcxStyle;
    cxStyle2: TcxStyle;
    cxStyle3: TcxStyle;
    cxStyle4: TcxStyle;
    cxStyle5: TcxStyle;
    cxStyle6: TcxStyle;
    cxStyle7: TcxStyle;
    cxStyle8: TcxStyle;
    cxStyle9: TcxStyle;
    cxStyle10: TcxStyle;
    cxStyle11: TcxStyle;
    cxStyle12: TcxStyle;
    cxStyle13: TcxStyle;
    cxStyle14: TcxStyle;
  end;

var
  CustomDrawTableViewDemoMainDM: TCustomDrawTableViewDemoMainDM;

implementation

{$R *.dfm}

end.
